Userland/Utilities/memstat.cpp

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (c) 2023, Liav A. <liavalb@hotmail.co.il>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/Format.h>
+#include <AK/JsonObject.h>
+#include <AK/JsonValue.h>
+#include <AK/NumberFormat.h>
+#include <AK/String.h>
+#include <LibCore/ArgsParser.h>
+#include <LibCore/File.h>
+#include <LibCore/System.h>
+#include <LibMain/Main.h>
+#include <unistd.h>
+
+static inline u64 page_count_to_bytes(size_t count)
+{
+    return count * 4096;
+}
+
+ErrorOr<int> serenity_main(Main::Arguments arguments)
+{
+    TRY(Core::System::unveil("/sys/kernel/memstat", "r"));
+    TRY(Core::System::unveil(nullptr, nullptr));
+
+    TRY(Core::System::pledge("stdio rpath"));
+
+    bool flag_human_readable = false;
+    Core::ArgsParser args_parser;
+    args_parser.add_option(flag_human_readable, "Print human-readable sizes", "human-readable", 'H');
```

Comment:
minor nit: Other utilities generally use `-h` to show human-readable sizes (unless there is a utility I am unaware of that you are trying to mimic?)